guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -4,43 +4,31 @@
 {ProjectName} can import content from the Red{nbsp}Hat Content Delivery Network (CDN).
 {Project} requires a Red{nbsp}Hat subscription manifest to find, access, and download content from the corresponding repositories.
 You must have a Red{nbsp}Hat subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
-All subscription information is available in your Red Hat Customer Portal account.
+All subscription information is available on the {RHCloud}.
 
 Use this chapter to import a Red{nbsp}Hat subscription manifest and manage the manifest within the {ProjectWebUI}.
 
 .Subscription allocations and organizations
-
 You can manage more than one organization if you have more than one subscription allocation.
 {Project} requires a single allocation for each organization configured in {ProjectServer}.
-The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red Hat accounts.
+The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red{nbsp}Hat accounts.
 
 .Future-dated subscriptions
-
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
-.Prerequisites
-* Ensure you have a Red{nbsp}Hat subscription manifest.
-ifdef::orcharhino[]
-You can create and download your manifest in the https://access.redhat.com/management/subscription_allocations[Red Hat Customer Portal].
-endif::[]
-ifndef::orcharhino[]
-** If your {Project} is connected, use the {RHCloud} to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
```

Comment:
The [SAT-29714](https://issues.redhat.com/browse/SAT-29714) Jira issue requested these links to be replaced by procedures. Where are those procedures?

If you plan to add them in another PR, can we keep these links for now?

Reply:
Yes